docs/wiki/ALZ-Contribution-Guide.md

```diff
@@ -0,0 +1,183 @@
+    1. Understand your issue :memo:
+    1. Reproduce the behavior :computer:
+    1. Provide evidence :mag_right:
+    1. Optionally, let us know if you would like to contribute a fix via a [**Pull Request**](https://github.com/Azure/Enterprise-Scale/pulls) :wrench:
+
+### Feature requests
+
+We understand that our solutions are going to always be a work in progress, and that customers will need and want to request new features.
+This is where you can really make a difference to how the solution is shaped for our community.
+
+If you have an idea you would like to be considered for inclusion, please use the following process:
+
+1. Familiarize yourself with our [conceptual architecture](https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/#azure-landing-zone-conceptual-architecture), [design principles](https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-principles) and [design areas](https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-areas) to ensure the feature aligns with the Azure landing zone guidance.
+1. Check existing [**GitHub Issues**](https://github.com/Azure/Enterprise-Scale/issues) to see whether the issue has already been reported.
+    1. If the issue is **open**, add a comment rather than create a new one.
+    1. If the issue is **closed**, check whether the proposed fix resolves your issue.
+1. Report it via our [**GitHub Issues**](https://github.com/Azure/Enterprise-Scale/issues)
+1. Select `New issue` and use the `Feature request 🚀` template
+1. Ensure you fill out the template with as much information as possible, being sure to cover off what's needed for maintainers and the community to:
+    1. Understand your feature and how it aligns to our [conceptual architecture](https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/#azure-landing-zone-conceptual-architecture), [design principles](https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-principles) and [design areas](https://docs.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-areas) :memo:
+    1. Optionally, let us know if you would like to contribute by adding your requested feature via a [**Pull Request**](https://github.com/Azure/Enterprise-Scale/pulls) :wrench:
+
+> **IMPORTANT:** If you are proposing a change to any of the Azure landing zone guidance, please include a business case explaining why you feel this will benefit our community.
+
+### Report a security vulnerability
+
+Please see our [**security policy**](https://github.com/Azure/Enterprise-Scale/security/policy) for more information.
+
+### Working with ALZ Custom policies
+
+Policies in the Azure Landing Zone reference implementations and repository are custom to Azure environments. They are definitions which are recommended when working with ALZ landing zones.
```

Comment:
Is it worth explicitly calling out that all reference implementations use the same set of policies, and these are all mastered in (and sourced from) the Azure/Enterprise-Scale repository?

Reply:
Good call. Added a note